stable/traefik/values.yaml

```diff
@@ -42,6 +42,10 @@ service:
   #   key: value
 gzip:
   enabled: true
+accessLogs:
+  enabled: false
+  filePath: ""  # if empty, stdout is used
+  format: "common"  # choices are: common, json
```

Comment:
For consistency, please remove quotes. Throughout the rest of this file, quotes aren't used in scenarios where they're optional.